app/models/sales_log.rb

```diff
@@ -41,6 +41,14 @@ class SalesLog < Log
       .or(filter_by_id(param))
   }
   scope :filter_by_organisation, ->(org, _user = nil) { where(owning_organisation: org) }
+  scope :duplicate_logs, lambda { |log|
+    attrs_to_check = %w[purchid saledate age1 sex1 ecstat1 postcode_full]
```

Comment:
I prefer this indentation to the one in the equivalent block in lettings_log.rb, can you use that there too?